test/index.js

```diff
@@ -2,19 +2,16 @@
 
 var os = require('os');
 var path = require('path');
+var pipeline = require('stream').pipeline;
 
 var fs = require('graceful-fs');
-var miss = require('mississippi');
 var mock = require('jest-mock');
 var expect = require('expect');
 var rimraf = require('rimraf');
+var Readable = require('streamx').Readable;
 
 var mkdirpStream = require('../');
 
-var pipe = miss.pipe;
-var from = miss.from;
-var concat = miss.concat;
```

Comment:
This should be a `new Writeable()` and still used in tests, see my other comment about ending on a writeable.

Reply:
Actually each test needs it's own `new Writable()` otherwise we get:
> MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [Writable]. Use emitter.setMaxListeners() to increase limit